src/client.ts

```diff
@@ -298,6 +298,23 @@ export class Client extends EventEmitter {
     }
   }
 
+  /**
+   * Queries the code corresponding to a contract
+   *
+   * @param contractAddress Contract address returned by deploy.
+   * @return Uint8Array The corresponding contract code
+   */
+  async getCodeAsync(contractAddress: Address): Promise<any> {
```

Comment:
Should be returning Promise<Uint8Array|null>

Reply:
Fixed 👍